src/persist/persist.ts

```diff
@@ -25,21 +25,29 @@ export async function persist(
   clientID: ClientID,
   memdag: dag.Store,
   perdag: dag.Store,
+  skipClientExists?: 'skip',
```

Comment:
I think its best not to skip this in tests.  

It's straightforward to set up the expected client using the test helpers in `clients-test-helpers.ts`.

Reply:
Will do.